### ISSUE TITLE
modid missing values and duplicate values

### ISSUE THREAD

**Issue by helixhamin** [DarkstarProject/darkstar#4896]
**_I have:_** Checked the status.lua, and found modid has gaps, and has some duplicate ids assigned to different mods

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:**  N/A

**_Source Branch_** (master/stable) **:** Master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Check status.lua modids for the following:
Missing modids (possibly can be used as reserve IDs):
53
74-79
123-126
138-143
152-159
190-223
239
256
260-287
290
297
377-379
571-825
830
888

Duplicate modid:
ROLL_TACTICIANS | 873
AUGMENTS_FEINT | 873

ROLL_ALLIES | 874
SNEAK_ATK_DEX | 874

**Comment by nfaletra** [comment]
Use modifier.h as your guide instead of status.lua. status.lua should match modifier.cpp.

Also: 

`// Weaponskill %damage modifiers
    // The following modifier should not ever be set, but %damage modifiers to weaponskills use the next 255 IDs (this modifier + the WSID)
    // For example, +10% damage to Chant du Cygne would be ID 570 + 225 (795)
    WEAPONSKILL_DAMAGE_BASE   = 570,`

**Comment by helixhamin** [comment]
@nfaletra 

Modifier.h shows:
AUGMENTS_FEINT            = 873, // Feint will give another -10 Evasion per merit level
ROLL_TACTICIANS           = 873, // Tracks totals
SNEAK_ATK_DEX             = 874, // % DEX boost to Sneak Attack (if gear mod, needs to be equipped on hit)
ROLL_ALLIES               = 874, // Tracks totals

status.lua shows:
AUGMENTS_FEINT                  = 873, -- Feint will give another -10 Evasion per merit level
ROLL_TACTICIANS                 = 873,
SNEAK_ATK_DEX                   = 874, -- % DEX boost to Sneak Attack (if gear mod, needs to be equipped on hit)
ROLL_ALLIES                     = 874,

Duplicates match.
I have not tested for all missing, but some random checks seem to match (aka, the missing values seem to also be missing in the modifier.h file)

**Comment by nfaletra** [comment]
Sorry, didn't mean to make it sound like I was saying you were wrong. The duplicates are definitely there. I was just mentioning that we do specific ws damage mods by using WEAPONSKILL_DAMAGE_BASE + WSID. So 571-795 are actually being used.

**Comment by helixhamin** [comment]
No, I understand, that is why I just listed what I did not see. I notice that the status.lua actually states the values are part of ws stuff. Basically, I am just trying to do some quick checks of items (excel spreadsheet), and that was just the numbers I noted. Every skipped number might be used by something else for all I know, I have not really had a chance to dig deeper. Just using this topic as a reference (I will change the remove those wsid numbers though)

**Comment by TeoTwawki** [comment]
even having to question it is a sign we need better documentation in there while ensuring both files mirror each other's mod lists fully. it being non sequential makes it harder to spot the gaps but organizing things into groups makes it easier on finding related mods, its really a mess. And re-sequencing is out of the question - we'd have to redo the entire item mods sql. So we'll just have to fix it once and then make dang sure any new changes are good and reviewed.
